src-gui/src/renderer/components/modal/introduction/slides/Slide02_ChooseAMaker.tsx

```diff
@@ -0,0 +1,26 @@
+import { Typography } from '@material-ui/core'
+import SlideTemplate from './SlideTemplate'
+import imagePath from '../../../../../assets/mockMakerSelection.webp'
+
+export default function Slide02_ChooseAMaker(props: slideProps) {
+    return (
+        <SlideTemplate
+            title="Choose a Maker"
+            stepLabel="Step 1"
+            {...props}
+            imagePath={imagePath}
+        >
+            <Typography variant="subtitle1">
+                Makers provide liquidity in XMR which can be exchanged for
```

Comment:
The text here can be made little bit more clear I'd say:
- Instead of "Makers provide liquidity" I'd say something like "Makers sell Monero in exchange for Bitcoin".
- I'd prefer not to abbreviate "minimum" / "maximum"
- I feel like "Makers can be found with" implies the user has to manually take some action but connecting to the public registry is done automatically and we are moving towards requiring very little user intervention for rendezvous discovery as well (#257). I'd say something along the lines of "To discover makers the GUI automatically connects to:..."